app/page.tsx

```diff
@@ -24,20 +60,48 @@ const HomePage = () => {
               type="text"
               id="ListInput"
               placeholder="What needs to be done?"
+              value={inputValue}
+              onChange={(e: ChangeEvent<HTMLInputElement>) => setInputValue(e.target.value)}
+              onKeyDown={handleKeyDown}
               className="bg-[#374151] text-[#e0e0e0] placeholder-[#a0a0a0] h-[50px] w-full p-[10px] text-lg outline-none border border-[#4b5563] focus:border-2 focus:border-[#8b5cf6] transition"
             />
-            <div className="element_container w-full" id="elementContainer"></div>
+            <div className="element_container w-full" id="elementContainer">
+            {items.map((item, index) => (
```

Comment:
make sense for check `items` count, if more then 0 then render DOM, or skip and return `null`

```
items.count > 1 ? (DOM) : null
```